crates/isahc_http_client/src/isahc_http_client.rs

```diff
@@ -54,6 +63,7 @@ impl HttpClient for IsahcHttpClient {
             let mut builder = isahc::Request::builder()
                 .method(parts.method)
                 .uri(parts.uri)
+                .proxy(self.proxy.clone())
```

Comment:
BTW, I dont know if this is needed, since we call `client_with_proxy.send_async(req)` later.